### ISSUE TITLE
JSON field is string

### ISSUE THREAD

**Issue by bitrut** [CanopyTax/asyncpgsa#44]
Having field defined this way `Column('abc', JSON)`, I have to use `json.loads` to get dict:
```
rows = await pg.fetch(sql)
async for row in rows:
    print(json.loads(row.abc))
```
`row.abc` is string

**Comment by nhumrich** [comment]
Yes. That is correct. It might be considered premature to json load a json field that you might or might not want into a dict. Also, you might want to use ujson or something faster than standard json module.

That being said, I am not opposed to json being automatically loaded, maybe with a flag in the pool object? Your welcome to write a pull request that will implement this feature.

**Comment by mackeyja92** [comment]
I believe loading JSON to be a feature of the ORM. Looking at the readme of this project it seems to only support SA core and not the ORM.  [The docs](http://docs.sqlalchemy.org/en/latest/dialects/postgresql.html#sqlalchemy.dialects.postgresql.JSON) for SA mention you can specify  `json_serializer` and  `json_deserializer` functions in the call to `create_engine`. I believe all that would need to be done is to create an engine interface for the ORM and you would be able to load the objects from the JSON automatically. 

It is possible to do it automatically in here though but I am not sure if that would cause compatibility issues later when integrating with the ORM.

**Comment by achimnol** [comment]
This issue is analogous to #45 .

**Comment by pierec** [comment]
Hi! 0.18 changelog says that:

> (...) In that case, it will actuall json dumps and loads automatically for you, which will break if you did it manually in your own code.

but I can't seem to get that to work like that with the default dialect settings. Is there something extra that needs to be done to get JSON auto-loaded and auto-dumped?

**Comment by nhumrich** [comment]
hmm, this might not actually work... I will need to circle back on that.

**Comment by kamikaze** [comment]
any news on this?

**Comment by nhumrich** [comment]
This is a bug, that requires a lot more work to get working. For now you'll just have to json decode json feilds I guess. Pr's welcome ;) 

I was premature in saying json decoding will work for you in the release notes. I misunderstood some things.
